backend/src/models/ClientServer.js

```
@@ -0,0 +1,529 @@
+
+      return clientServer;
+   }
+
+   /**
+    * Convert to safe API response
+    * @returns {Object} ClientServer without sensitive data
+    */
+   toApiResponse() {
+      const response = super.toApiResponse();
+      // Never include the secret hash in API responses
+      delete response.client_secret_hash;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)